src/Bindings/DeprecatedBindings.cpp

```diff
@@ -293,20 +299,32 @@ static int tolua_cWorld_SetSignLines(lua_State * tolua_S)
 		!tolua_iscppstring(LuaState, 8, 0, &tolua_err) ||
 		!tolua_isusertype (LuaState, 9, "cPlayer", 1, &tolua_err) ||
 		!tolua_isnoobj    (LuaState, 10, &tolua_err)
+		*/
 		)
 		goto tolua_lerror;
```

Comment:
Return immediately, the `CheckParamXYZ()` functions already do output the error message so there's no need to do anything more.